Create fr_fr.json

Neshaiy opened the pull request:
Adding French language support to Hideaway+

Commit:
Create fr_fr.json

Adding French language support to Hideaway+

oliviathevampire commented on src/main/resources/assets/hideaway_plus/lang/fr_fr.json:
```diff
@@ -0,0 +1,36 @@
+    "text.config.hp-config.option.autoSell": "Remplir automatiquement l'écran de la boutique",
+    "text.config.hp-config.option.autoEnableEditor": "Activer automatiquement l'éditeur",
+    "text.config.hp-config.option.jukebox": "Utiliser le jukebox",
+    "text.config.hp-config.option.noAmbientSounds": "Autoriser les sons ambiants",
+    "text.config.hp-config.option.noActivitySongs": "Autoriser les musiques d'activité"
```
My french friend said it should be Désactiver instead of Autoriser

Commit:
Merge branch 'Voxxin:main' into main

Commit:
Update fr_fr.json

Following the correction of "en_us.json" I corrected the translation accordingly
